src/library/scala/collection/immutable/Range.scala

```diff
@@ -400,18 +400,17 @@ extends scala.collection.AbstractSeq[Int]
    *  equals contract.
    */
 
-  override def toString() = {
-    val endStr =
-      if (numRangeElements > Range.MAX_PRINT || (!isEmpty && numRangeElements < 0)) ", ... )" else ")"
-    take(Range.MAX_PRINT).mkString("Range(", ", ", endStr)
+  override def toString = {
+    val preposition = if (isInclusive) "to" else "until"
+    f"$start $preposition $end${
+      if (step == 1) "" else f" by $step"
+    }"
```

Comment:
Just a finger twitch. At first I wrote `f"$start%d"`, I think. But it's interesting to hear the JS consequences. f-interpolator needs to special case if it's just appending strings. It already detects constant results.

For some reason, I really want the multiline interpolated expression to be readable, but that hasn't actually happened yet.